dynawo/sources/Models/Modelica/Dynawo/Examples/HVDC/HVDC.mo

```diff
@@ -0,0 +1,85 @@
+within Dynawo.Examples.HVDC;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model HVDC "HVDC link connected to two infinite buses"
+
+  import Dynawo;
```

Comment:
In order to run on my side I need import Modelica here

Reply:
Done